### ISSUE TITLE
android docker?

### ISSUE THREAD

**Issue by thedavidmeister** [holochain/holochain-rust#110]
can we test/build for android somehow without needing the full android studio?

i'm finding that it's quite a big download and takes a while to setup, with many steps along the way :/

from #66

**Comment by jamesray1** [comment]
This looks outdated.

https://github.com/holochain/holochain-rust/blob/958907ee618c2bc3c6b9c01c4ea6989f0376b6de/doc/holochain_101/src/building_for_android.md

**Comment by thedavidmeister** [comment]
no, those instructions start with "install android studio", i'm wondering if we can investigate a solution that allows us to build and test without such an expensive setup

**Comment by jamesray1** [comment]
Ah, sorry. I found this:

https://developer.android.com/studio/build/building-cmdline

(which was found from here:

https://stackoverflow.com/questions/32643297/how-to-make-an-android-app-without-using-android-studio

(which in turn was found from here:

https://duckduckgo.com/?q=can+we+test%2Fbuild+for+android+somehow+without+needing+the+full+android+studio%3F&t=ffab&atb=v168-1&ia=web
))
